etcdmain/etcd.go

```diff
@@ -145,6 +145,9 @@ func startEtcd(cfg *config) (<-chan struct{}, error) {
 	}
 	plns := make([]net.Listener, 0)
 	for _, u := range cfg.lpurls {
+		if u.Scheme == "http" && !cfg.peerTLSInfo.Empty() {
+			plog.Warningf("peer url %s does not use TLS when peer TLS files are presented", u.String())
```

Comment:
The scheme of peer url is http while peer key/cert files are presented. Ignored key/cert files.